### PR TITLE
PLAT-1224: Fix tooltip padding

### PR DESCRIPTION
**Pull request by mihaelpalfi**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/PLAT-1224)

#### Description
As per JIRA ticket

#### ChangeLOG

- [ ] I have added all notable deployment/development environment (onprem!) changes to [CHANGELOG.md](https://github.com/reciprocity/zen-ui/blob/main/README.md)

#### Useful links
[Zen UI readme](https://github.com/reciprocity/zen-ui/blob/main/README.md)

[PR review guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/pull_request_reviews.md)

[Git commit guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/commit_guidelines.md)

### CODE REVIEW

File: src/components/zen-tooltip/zen-tooltip.stories.mdx [hunk 1]
```diff
@@ -2,203 +2,122 @@ import { html } from 'lit-html';
...
       <div>
         <zen-button label="Info" variant="secondary"></zen-button>
         <zen-tooltip
           always-visible="true"
           position="top"
           variant="light"
-          label="This is a information tooltip."
-        ></zen-tooltip>
+        >
+            <zen-space padding="lg">
```
Comment (chavlji) [inline src/components/zen-tooltip/zen-tooltip.stories.mdx:83]: Imo both `zen-space` and `zen-text` should be a part of default content... It's very tidious to have to repeat this throughout the app all the time for each small tooltip.
Just make default slot content of `zen-text` then.
For more custom cases we can have a slot named `content`. If slot `content` is fulfilled, we wouldn't render `zen-space` and `zen-text` but just this slot instead.
That would provide high standardization along with high customization when needed.

**Review by chavlji — CHANGES_REQUESTED** [review]
Nice! It all looks good, I just have concerns regarding the convenience of such highly customizable tooltips.

File: src/components/zen-tooltip/zen-tooltip.stories.mdx (hunk 1 above)
Comment (mihaelpalfi) [reply src/components/zen-tooltip/zen-tooltip.stories.mdx:83]: Do we do this for the tooltip or in popover? Why do you think we  have to remove the tooltip?

Comment (chavlji) [reply src/components/zen-tooltip/zen-tooltip.stories.mdx:83]: In tooltip. Popover's content is so specific and custom that it's expected users should provide padding. But when it comes to tooltips, 99% of tooltips focus just on the text content and should look uniformed across the apps.

What do you mean by "remove the tooltip"? I just thought we should use the popover component for the tooltip container when it's ready. This way we don't have to calculate its position manually.

Comment (mihaelpalfi) [reply src/components/zen-tooltip/zen-tooltip.stories.mdx:83]: Updated that zen space wrappes default slot content. And added slot content for special contents.

**Review by chavlji — APPROVED** [review]

**Pull request merged**